src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/client/NodeClientTest.kt

```diff
@@ -32,7 +32,9 @@ class NodeClientTest {
   @Mock private lateinit var nodeApi: NodoApi
 
   @InjectMocks private lateinit var nodeClient: NodeClient
-
+  companion object {
+    const val CLIENT_ID_CLOSE_PAYMENT: String = "ecomm"
```

Comment:
Is the value of the client id already a contract between us and Nodo?
Otherwise I would avoid abbreviating it
```suggestion
    const val CLOSE_PAYMENT_CLIENT_ID: String = "eCommerce"
```